textattack/constraints/semantics/sentence_encoders/sentence_encoder.py

```diff
@@ -153,7 +153,7 @@ def _score_list(self, starting_text, transformed_texts):
                 # If the embedding is not yet a tensor, make it one.
                 starting_embedding = torch.tensor(embeddings[0]).to(utils.device)
 
-            if isinstance(embeddings, list):
+            if isinstance(embeddings[1], torch.Tensor):
```

Comment:
why?

Reply:
I don't know why we do `instance(embeddings, list)` check. But to do `torch.stack`, the elements themselves need to be a Tensor. I'm just here assuming that checking `embeddings[1]` is good enough since it's batch output. If you run it with `bert` sentence encoder, I'm see that `embedding` is a list of `np.array`. In that case, we should just do `torch.tesnor`.

Reply:
ok I think it's bc the old sentence encoder outputted a list